wallet/src/components/BridgeComponent/index.tsx

```diff
@@ -145,6 +146,15 @@ const BridgeComponent = (props: any) => {
     return true;
   }
 
+  useEffect(() => {
```

Comment:
Not sure if adding it here and refreshing based on `txType` is a good experience. It means that this pops up every time you click on the toggle between `Deposit` and `Withdraw` even if you are not actually performing any of the operations